easy-ui-react/src/Drawer/Drawer.mdx

```diff
@@ -0,0 +1,159 @@
+
+Alternatively, a `<Drawer />` can be wrapped in a `<DrawerContainer />` to entirely control the triggering process. See the Custom Triggers story below.
+
+### State management
+
+By default, a `<Drawer />` manages its own open state.
+
+It can be made to be a controlled component by using the `isOpen` and `onOpenChange` prop on `<Drawer.Trigger />`.
+
+It can be made to open by default using the `defaultOpen` prop on `<Drawer.Trigger />`.
+
+Use the `useDrawerTrigger()` hook inside a Drawer component to access the drawer state. Use the drawer trigger state to determine if the drawer `isOpen`. The drawer trigger state can also be used to programmatically `close()` the drawer.
+
+### Focus containment
+
+A `<Drawer />` traps focus until it's closed. When a drawer is closed, it returns focus to the element that triggered the drawer.
+
+## Standalone Example
+
+In its simplest form, a `<Drawer />` requires a triggering element and a `<Drawer />` that renders a body.
+
+<Canvas of={DrawerStories.Standalone} />
+
+## Banner and Tabs Example
+
+A `<Drawer />` offers several components to customize the experience. This example uses all available components.
+
+Refer to the property tables at the bottom of the page for full documentation.
+
+<Canvas of={DrawerStories.BannerAndTabs} />
+
+## Nondismissable
+
+By default, `<Drawer />`s are dismissable, meaning a user can exit the drawer without performing an action. Exiting can be done by clicking on the underlay behind the drawer, pressing the `Esc` key, or clicking the close icon.
+
+To override that behavior, set `isDismissable` to `false` on the `<Drawer.Trigger />`.
+
+<Canvas of={DrawerStories.Nondismissable} />
+
+<Controls of={DrawerStories.Nondismissable} />
+
+## Custom Triggers
+
+A `<Drawer />` can be wrapped in a `<DrawerContainer />` to entirely control the triggering process.
+
+A `<DrawerContainer />` renders a single `<Drawer />` as a child. If no child is provided, the `<DrawerContainer />` renders itself as closed. Use the `onDismiss` prop to know when the drawer closes.
+
+<Canvas of={DrawerStories.MenuTrigger} />
+
+## Properties
+
+### Drawer.Trigger
+
+<ArgTypes of={Drawer.Trigger} />
+
+### Drawer
+
+<ArgTypes of={Drawer} />
+
+### Drawer.Header
+
+<ArgTypes of={Drawer.Header} />
+
+### Drawer.Body
+
+<ArgTypes of={Drawer.Body} />
+
+### Drawer.Body
+
+<ArgTypes of={Drawer.Body} />
+
+### Drawer.Title
+
+<ArgTypes of={Drawer.Title} />
+
+### Drawer.Banner
+
+<ArgTypes of={Drawer.Banner} />
+
+### Drawer.BanneredContentArea
+
+<ArgTypes of={Drawer.BanneredContentArea} />
+
+### Drawer.StandaloneContentArea
+
+<ArgTypes of={Drawer.StandaloneContentArea} />
+
+### Drawer.CloseButton
+
+<ArgTypes of={Drawer.CloseButton} />
```

Comment:
On the storybook, this doesn't show anything, is this intentional?

Reply:
yeah it has no props. i'll remove it since it's confusing